src/com/galactanet/gametable/events/PogMenuRenderEvent.java

```diff
@@ -0,0 +1,30 @@
+package com.galactanet.gametable.events;
+
+import com.galactanet.gametable.Pog;
+
+import javax.swing.JPopupMenu;
+
+public class PogMenuRenderEvent {
```

Comment:
Also we might need to expand what goes into here for other menu items. YAGNI for now